bigquery_etl/public_data/publish_gcs_metadata.py

```diff
@@ -10,8 +10,8 @@
 from google.cloud import storage
 from itertools import groupby
 
-from ..parse_metadata import Metadata
-from ..util import standard_args
+from bigquery_etl.metadata.parse_metadata import Metadata
```

Comment:
I thought it would make sense to move all the metadata scripts (since there are now 3 of them) to a new `metadata` directory to keep things cleaner.